src/Data/Bin/Properties.agda

```diff
@@ -20,6 +20,8 @@ open import Data.Nat.Solver
 open import Data.Product using (_,_; proj₁; proj₂; ∃)
 open import Data.Sum using (_⊎_; inj₁; inj₂)
 open import Function using (_∘_; _$_; id; const)
+open import Function.Definitions using (Injective)
+open import Function.Definitions.Core2 using (Surjective)
```

Comment:
I fail with setting
```
open import Function.Definitions using (Injective; Surjective)
```
Because there are declared
```
toℕ-injective :  Injective {A = Bin} {B = ℕ} _≡_ _≡_ toℕ
...
toℕ-surjective :  Surjective {B = ℕ} _≡_ toℕ
```
, and the modules Function.Definitions and Function.Definitions.Core2 
have different parameters. 
All this leads to the report of
"ℕ !=< Bin of type Set
 when checking that the expression toℕ has type Rel ℕ _ℓ₂_415
"
for the declaration ``toℕ-surjective :  Surjective {B = ℕ} _≡_ toℕ``
And I do not guess what arguments needs the line ``toℕ-surjective : ...`` 
Please, advise.